src/main/drivers/adc.c

```diff
@@ -70,6 +70,13 @@ ADCDevice adcDeviceByInstance(ADC_TypeDef *instance)
 
 uint16_t adcGetChannel(uint8_t channel)
 {
+    if (adcOperatingConfig[4].enabled) {
```

Comment:
Should be inside the `#ifdef DEBUG_ADC_CHANNELS`. (or better yet, convert the whole shebang to use`DEBUG_SET`).

Reply:
Meh... These are the only outputs from this mod. Leave it while evaluating.

Reply:
Just listing things that will have to be fixed before this gets merged eventually. Forgotten debug output statements are probably the most common software bug that I come across when reviewing code, and they are very easily overlooked in QA...